packages/protocol/contracts/thirdparty/lido/BridgeableTokens.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+/// @author psirex
+/// @notice Contains the logic for validation of tokens used in the bridging process
+/// @dev Taken from https://github.com/lidofinance/lido-l2/blob/main/contracts/BridgeableTokens.sol
+contract BridgeableTokens {
```

Comment:
So all these contracts are not upgradable?

Reply:
Currently not upgradable. Will make them upgradable